packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -564,16 +564,14 @@ describe('Ceramic integration', () => {
             genesis: genesisCommit,
           },
         ])
-      ).rejects.toThrowError(
```

Comment:
I don't like losing this feedback to users who misuse the multiquery api.  All the changes to this test file make me nervous.  Please see my comment on the multiquery code for an idea of how to avoid this while still handling the uncaught exception issue you were hitting.